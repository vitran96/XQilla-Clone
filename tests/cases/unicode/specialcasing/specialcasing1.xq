declare boundary-space preserve;
<result>
<code_point value="223">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x00DF;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x00DF;'))}</upper>
</code_point>
<code_point value="304">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x0130;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x0130;'))}</upper>
</code_point>
<code_point value="64256">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB00;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB00;'))}</upper>
</code_point>
<code_point value="64257">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB01;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB01;'))}</upper>
</code_point>
<code_point value="64258">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB02;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB02;'))}</upper>
</code_point>
<code_point value="64259">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB03;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB03;'))}</upper>
</code_point>
<code_point value="64260">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB04;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB04;'))}</upper>
</code_point>
<code_point value="64261">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB05;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB05;'))}</upper>
</code_point>
<code_point value="64262">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB06;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB06;'))}</upper>
</code_point>
<code_point value="1415">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x0587;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x0587;'))}</upper>
</code_point>
<code_point value="64275">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB13;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB13;'))}</upper>
</code_point>
<code_point value="64276">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB14;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB14;'))}</upper>
</code_point>
<code_point value="64277">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB15;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB15;'))}</upper>
</code_point>
<code_point value="64278">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB16;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB16;'))}</upper>
</code_point>
<code_point value="64279">
<lower>{fn:string-to-codepoints(fn:lower-case('&#xFB17;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#xFB17;'))}</upper>
</code_point>
<code_point value="329">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x0149;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x0149;'))}</upper>
</code_point>
<code_point value="912">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x0390;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x0390;'))}</upper>
</code_point>
<code_point value="944">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x03B0;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x03B0;'))}</upper>
</code_point>
<code_point value="496">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x01F0;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x01F0;'))}</upper>
</code_point>
<code_point value="7830">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1E96;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1E96;'))}</upper>
</code_point>
<code_point value="7831">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1E97;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1E97;'))}</upper>
</code_point>
<code_point value="7832">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1E98;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1E98;'))}</upper>
</code_point>
<code_point value="7833">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1E99;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1E99;'))}</upper>
</code_point>
<code_point value="7834">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1E9A;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1E9A;'))}</upper>
</code_point>
<code_point value="8016">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F50;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F50;'))}</upper>
</code_point>
<code_point value="8018">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F52;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F52;'))}</upper>
</code_point>
<code_point value="8020">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F54;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F54;'))}</upper>
</code_point>
<code_point value="8022">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F56;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F56;'))}</upper>
</code_point>
<code_point value="8118">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FB6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FB6;'))}</upper>
</code_point>
<code_point value="8134">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FC6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FC6;'))}</upper>
</code_point>
<code_point value="8146">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FD2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FD2;'))}</upper>
</code_point>
<code_point value="8147">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FD3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FD3;'))}</upper>
</code_point>
<code_point value="8150">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FD6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FD6;'))}</upper>
</code_point>
<code_point value="8151">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FD7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FD7;'))}</upper>
</code_point>
<code_point value="8162">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FE2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FE2;'))}</upper>
</code_point>
<code_point value="8163">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FE3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FE3;'))}</upper>
</code_point>
<code_point value="8164">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FE4;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FE4;'))}</upper>
</code_point>
<code_point value="8166">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FE6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FE6;'))}</upper>
</code_point>
<code_point value="8167">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FE7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FE7;'))}</upper>
</code_point>
<code_point value="8182">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FF6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FF6;'))}</upper>
</code_point>
<code_point value="8064">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F80;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F80;'))}</upper>
</code_point>
<code_point value="8065">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F81;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F81;'))}</upper>
</code_point>
<code_point value="8066">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F82;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F82;'))}</upper>
</code_point>
<code_point value="8067">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F83;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F83;'))}</upper>
</code_point>
<code_point value="8068">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F84;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F84;'))}</upper>
</code_point>
<code_point value="8069">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F85;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F85;'))}</upper>
</code_point>
<code_point value="8070">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F86;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F86;'))}</upper>
</code_point>
<code_point value="8071">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F87;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F87;'))}</upper>
</code_point>
<code_point value="8072">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F88;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F88;'))}</upper>
</code_point>
<code_point value="8073">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F89;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F89;'))}</upper>
</code_point>
<code_point value="8074">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8A;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8A;'))}</upper>
</code_point>
<code_point value="8075">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8B;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8B;'))}</upper>
</code_point>
<code_point value="8076">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8C;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8C;'))}</upper>
</code_point>
<code_point value="8077">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8D;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8D;'))}</upper>
</code_point>
<code_point value="8078">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8E;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8E;'))}</upper>
</code_point>
<code_point value="8079">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F8F;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F8F;'))}</upper>
</code_point>
<code_point value="8080">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F90;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F90;'))}</upper>
</code_point>
<code_point value="8081">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F91;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F91;'))}</upper>
</code_point>
<code_point value="8082">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F92;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F92;'))}</upper>
</code_point>
<code_point value="8083">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F93;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F93;'))}</upper>
</code_point>
<code_point value="8084">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F94;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F94;'))}</upper>
</code_point>
<code_point value="8085">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F95;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F95;'))}</upper>
</code_point>
<code_point value="8086">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F96;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F96;'))}</upper>
</code_point>
<code_point value="8087">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F97;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F97;'))}</upper>
</code_point>
<code_point value="8088">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F98;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F98;'))}</upper>
</code_point>
<code_point value="8089">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F99;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F99;'))}</upper>
</code_point>
<code_point value="8090">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9A;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9A;'))}</upper>
</code_point>
<code_point value="8091">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9B;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9B;'))}</upper>
</code_point>
<code_point value="8092">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9C;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9C;'))}</upper>
</code_point>
<code_point value="8093">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9D;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9D;'))}</upper>
</code_point>
<code_point value="8094">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9E;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9E;'))}</upper>
</code_point>
<code_point value="8095">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1F9F;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1F9F;'))}</upper>
</code_point>
<code_point value="8096">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA0;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA0;'))}</upper>
</code_point>
<code_point value="8097">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA1;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA1;'))}</upper>
</code_point>
<code_point value="8098">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA2;'))}</upper>
</code_point>
<code_point value="8099">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA3;'))}</upper>
</code_point>
<code_point value="8100">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA4;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA4;'))}</upper>
</code_point>
<code_point value="8101">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA5;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA5;'))}</upper>
</code_point>
<code_point value="8102">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA6;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA6;'))}</upper>
</code_point>
<code_point value="8103">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA7;'))}</upper>
</code_point>
<code_point value="8104">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA8;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA8;'))}</upper>
</code_point>
<code_point value="8105">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FA9;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FA9;'))}</upper>
</code_point>
<code_point value="8106">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAA;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAA;'))}</upper>
</code_point>
<code_point value="8107">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAB;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAB;'))}</upper>
</code_point>
<code_point value="8108">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAC;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAC;'))}</upper>
</code_point>
<code_point value="8109">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAD;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAD;'))}</upper>
</code_point>
<code_point value="8110">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAE;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAE;'))}</upper>
</code_point>
<code_point value="8111">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FAF;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FAF;'))}</upper>
</code_point>
<code_point value="8115">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FB3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FB3;'))}</upper>
</code_point>
<code_point value="8124">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FBC;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FBC;'))}</upper>
</code_point>
<code_point value="8131">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FC3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FC3;'))}</upper>
</code_point>
<code_point value="8140">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FCC;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FCC;'))}</upper>
</code_point>
<code_point value="8179">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FF3;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FF3;'))}</upper>
</code_point>
<code_point value="8188">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FFC;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FFC;'))}</upper>
</code_point>
<code_point value="8114">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FB2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FB2;'))}</upper>
</code_point>
<code_point value="8116">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FB4;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FB4;'))}</upper>
</code_point>
<code_point value="8130">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FC2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FC2;'))}</upper>
</code_point>
<code_point value="8132">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FC4;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FC4;'))}</upper>
</code_point>
<code_point value="8178">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FF2;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FF2;'))}</upper>
</code_point>
<code_point value="8180">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FF4;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FF4;'))}</upper>
</code_point>
<code_point value="8119">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FB7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FB7;'))}</upper>
</code_point>
<code_point value="8135">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FC7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FC7;'))}</upper>
</code_point>
<code_point value="8183">
<lower>{fn:string-to-codepoints(fn:lower-case('&#x1FF7;'))}</lower>
<upper>{fn:string-to-codepoints(fn:upper-case('&#x1FF7;'))}</upper>
</code_point>
</result>
