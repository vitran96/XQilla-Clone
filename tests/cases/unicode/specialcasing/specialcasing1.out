<result>
<code_point value="223">
<lower>223</lower>
<upper>83 83</upper>
</code_point>
<code_point value="304">
<lower>105 775</lower>
<upper>304</upper>
</code_point>
<code_point value="64256">
<lower>64256</lower>
<upper>70 70</upper>
</code_point>
<code_point value="64257">
<lower>64257</lower>
<upper>70 73</upper>
</code_point>
<code_point value="64258">
<lower>64258</lower>
<upper>70 76</upper>
</code_point>
<code_point value="64259">
<lower>64259</lower>
<upper>70 70 73</upper>
</code_point>
<code_point value="64260">
<lower>64260</lower>
<upper>70 70 76</upper>
</code_point>
<code_point value="64261">
<lower>64261</lower>
<upper>83 84</upper>
</code_point>
<code_point value="64262">
<lower>64262</lower>
<upper>83 84</upper>
</code_point>
<code_point value="1415">
<lower>1415</lower>
<upper>1333 1362</upper>
</code_point>
<code_point value="64275">
<lower>64275</lower>
<upper>1348 1350</upper>
</code_point>
<code_point value="64276">
<lower>64276</lower>
<upper>1348 1333</upper>
</code_point>
<code_point value="64277">
<lower>64277</lower>
<upper>1348 1339</upper>
</code_point>
<code_point value="64278">
<lower>64278</lower>
<upper>1358 1350</upper>
</code_point>
<code_point value="64279">
<lower>64279</lower>
<upper>1348 1341</upper>
</code_point>
<code_point value="329">
<lower>329</lower>
<upper>700 78</upper>
</code_point>
<code_point value="912">
<lower>912</lower>
<upper>921 776 769</upper>
</code_point>
<code_point value="944">
<lower>944</lower>
<upper>933 776 769</upper>
</code_point>
<code_point value="496">
<lower>496</lower>
<upper>74 780</upper>
</code_point>
<code_point value="7830">
<lower>7830</lower>
<upper>72 817</upper>
</code_point>
<code_point value="7831">
<lower>7831</lower>
<upper>84 776</upper>
</code_point>
<code_point value="7832">
<lower>7832</lower>
<upper>87 778</upper>
</code_point>
<code_point value="7833">
<lower>7833</lower>
<upper>89 778</upper>
</code_point>
<code_point value="7834">
<lower>7834</lower>
<upper>65 702</upper>
</code_point>
<code_point value="8016">
<lower>8016</lower>
<upper>933 787</upper>
</code_point>
<code_point value="8018">
<lower>8018</lower>
<upper>933 787 768</upper>
</code_point>
<code_point value="8020">
<lower>8020</lower>
<upper>933 787 769</upper>
</code_point>
<code_point value="8022">
<lower>8022</lower>
<upper>933 787 834</upper>
</code_point>
<code_point value="8118">
<lower>8118</lower>
<upper>913 834</upper>
</code_point>
<code_point value="8134">
<lower>8134</lower>
<upper>919 834</upper>
</code_point>
<code_point value="8146">
<lower>8146</lower>
<upper>921 776 768</upper>
</code_point>
<code_point value="8147">
<lower>8147</lower>
<upper>921 776 769</upper>
</code_point>
<code_point value="8150">
<lower>8150</lower>
<upper>921 834</upper>
</code_point>
<code_point value="8151">
<lower>8151</lower>
<upper>921 776 834</upper>
</code_point>
<code_point value="8162">
<lower>8162</lower>
<upper>933 776 768</upper>
</code_point>
<code_point value="8163">
<lower>8163</lower>
<upper>933 776 769</upper>
</code_point>
<code_point value="8164">
<lower>8164</lower>
<upper>929 787</upper>
</code_point>
<code_point value="8166">
<lower>8166</lower>
<upper>933 834</upper>
</code_point>
<code_point value="8167">
<lower>8167</lower>
<upper>933 776 834</upper>
</code_point>
<code_point value="8182">
<lower>8182</lower>
<upper>937 834</upper>
</code_point>
<code_point value="8064">
<lower>8064</lower>
<upper>7944 921</upper>
</code_point>
<code_point value="8065">
<lower>8065</lower>
<upper>7945 921</upper>
</code_point>
<code_point value="8066">
<lower>8066</lower>
<upper>7946 921</upper>
</code_point>
<code_point value="8067">
<lower>8067</lower>
<upper>7947 921</upper>
</code_point>
<code_point value="8068">
<lower>8068</lower>
<upper>7948 921</upper>
</code_point>
<code_point value="8069">
<lower>8069</lower>
<upper>7949 921</upper>
</code_point>
<code_point value="8070">
<lower>8070</lower>
<upper>7950 921</upper>
</code_point>
<code_point value="8071">
<lower>8071</lower>
<upper>7951 921</upper>
</code_point>
<code_point value="8072">
<lower>8064</lower>
<upper>7944 921</upper>
</code_point>
<code_point value="8073">
<lower>8065</lower>
<upper>7945 921</upper>
</code_point>
<code_point value="8074">
<lower>8066</lower>
<upper>7946 921</upper>
</code_point>
<code_point value="8075">
<lower>8067</lower>
<upper>7947 921</upper>
</code_point>
<code_point value="8076">
<lower>8068</lower>
<upper>7948 921</upper>
</code_point>
<code_point value="8077">
<lower>8069</lower>
<upper>7949 921</upper>
</code_point>
<code_point value="8078">
<lower>8070</lower>
<upper>7950 921</upper>
</code_point>
<code_point value="8079">
<lower>8071</lower>
<upper>7951 921</upper>
</code_point>
<code_point value="8080">
<lower>8080</lower>
<upper>7976 921</upper>
</code_point>
<code_point value="8081">
<lower>8081</lower>
<upper>7977 921</upper>
</code_point>
<code_point value="8082">
<lower>8082</lower>
<upper>7978 921</upper>
</code_point>
<code_point value="8083">
<lower>8083</lower>
<upper>7979 921</upper>
</code_point>
<code_point value="8084">
<lower>8084</lower>
<upper>7980 921</upper>
</code_point>
<code_point value="8085">
<lower>8085</lower>
<upper>7981 921</upper>
</code_point>
<code_point value="8086">
<lower>8086</lower>
<upper>7982 921</upper>
</code_point>
<code_point value="8087">
<lower>8087</lower>
<upper>7983 921</upper>
</code_point>
<code_point value="8088">
<lower>8080</lower>
<upper>7976 921</upper>
</code_point>
<code_point value="8089">
<lower>8081</lower>
<upper>7977 921</upper>
</code_point>
<code_point value="8090">
<lower>8082</lower>
<upper>7978 921</upper>
</code_point>
<code_point value="8091">
<lower>8083</lower>
<upper>7979 921</upper>
</code_point>
<code_point value="8092">
<lower>8084</lower>
<upper>7980 921</upper>
</code_point>
<code_point value="8093">
<lower>8085</lower>
<upper>7981 921</upper>
</code_point>
<code_point value="8094">
<lower>8086</lower>
<upper>7982 921</upper>
</code_point>
<code_point value="8095">
<lower>8087</lower>
<upper>7983 921</upper>
</code_point>
<code_point value="8096">
<lower>8096</lower>
<upper>8040 921</upper>
</code_point>
<code_point value="8097">
<lower>8097</lower>
<upper>8041 921</upper>
</code_point>
<code_point value="8098">
<lower>8098</lower>
<upper>8042 921</upper>
</code_point>
<code_point value="8099">
<lower>8099</lower>
<upper>8043 921</upper>
</code_point>
<code_point value="8100">
<lower>8100</lower>
<upper>8044 921</upper>
</code_point>
<code_point value="8101">
<lower>8101</lower>
<upper>8045 921</upper>
</code_point>
<code_point value="8102">
<lower>8102</lower>
<upper>8046 921</upper>
</code_point>
<code_point value="8103">
<lower>8103</lower>
<upper>8047 921</upper>
</code_point>
<code_point value="8104">
<lower>8096</lower>
<upper>8040 921</upper>
</code_point>
<code_point value="8105">
<lower>8097</lower>
<upper>8041 921</upper>
</code_point>
<code_point value="8106">
<lower>8098</lower>
<upper>8042 921</upper>
</code_point>
<code_point value="8107">
<lower>8099</lower>
<upper>8043 921</upper>
</code_point>
<code_point value="8108">
<lower>8100</lower>
<upper>8044 921</upper>
</code_point>
<code_point value="8109">
<lower>8101</lower>
<upper>8045 921</upper>
</code_point>
<code_point value="8110">
<lower>8102</lower>
<upper>8046 921</upper>
</code_point>
<code_point value="8111">
<lower>8103</lower>
<upper>8047 921</upper>
</code_point>
<code_point value="8115">
<lower>8115</lower>
<upper>913 921</upper>
</code_point>
<code_point value="8124">
<lower>8115</lower>
<upper>913 921</upper>
</code_point>
<code_point value="8131">
<lower>8131</lower>
<upper>919 921</upper>
</code_point>
<code_point value="8140">
<lower>8131</lower>
<upper>919 921</upper>
</code_point>
<code_point value="8179">
<lower>8179</lower>
<upper>937 921</upper>
</code_point>
<code_point value="8188">
<lower>8179</lower>
<upper>937 921</upper>
</code_point>
<code_point value="8114">
<lower>8114</lower>
<upper>8122 921</upper>
</code_point>
<code_point value="8116">
<lower>8116</lower>
<upper>902 921</upper>
</code_point>
<code_point value="8130">
<lower>8130</lower>
<upper>8138 921</upper>
</code_point>
<code_point value="8132">
<lower>8132</lower>
<upper>905 921</upper>
</code_point>
<code_point value="8178">
<lower>8178</lower>
<upper>8186 921</upper>
</code_point>
<code_point value="8180">
<lower>8180</lower>
<upper>911 921</upper>
</code_point>
<code_point value="8119">
<lower>8119</lower>
<upper>913 834 921</upper>
</code_point>
<code_point value="8135">
<lower>8135</lower>
<upper>919 834 921</upper>
</code_point>
<code_point value="8183">
<lower>8183</lower>
<upper>937 834 921</upper>
</code_point>
</result>
